lib/plugins/aws/package/compile/events/cloudFront.js

```diff
@@ -163,6 +163,16 @@ class AwsCompileCloudFrontEvents {
         ViewerProtocolPolicy: {
           enum: ['allow-all', 'redirect-to-https', 'https-only'],
         },
+        TrustedKeyGroups: {
+          anyOf: [
```

Comment:
Is that `anyOf` needed here? I think it's an unnecessary wrapper.

Reply:
Nice catch @pgrzesik, I forgot to remove it